app/util/Html.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Static util class for HTML related operations.
+ */
+Ext.define('CpsiMapview.util.Html', {
+
+    statics: {
+        /**
+         * Add events to HTML elements in a cross browser compatible manor.
+         *
+         * Credits: http://stackoverflow.com/a/21140463
+         *
+         * @param {DOMElement} htmlElement The element to attach the event to.
+         * @param {String} eventName The name of the event to attach.
+         * @param {Function} eventFunction The handler for the `eventName` on
+         *     the passed `htmlElement`.
+         */
+        addEvent: function (htmlElement, eventName, eventFunction) {
```

Comment:
I presume ExtJS doesn't already have a function for this that could be used?

Reply:
As far as I know not, but I'll double check.

Reply:
Guess if there was it would be part of https://docs.sencha.com/extjs/6.2.0/classic/Ext.dom.Element.html ?

Reply:
Probably yes. I had a quick look and it seems `Ext.Dom.Element` has the normal `on` event binding method. Maybe it is worth giving this a try so we could get rid of the HTML util class.